src/huggingface_hub/commands/jobs.py

```diff
@@ -0,0 +1,471 @@
+# Copyright 2025 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Contains commands to interact with jobs on the Hugging Face Hub.
+
+Usage:
+    # run a job
+    huggingface-cli jobs run image command
+"""
+
+import io
+import json
+import os
+import re
+from argparse import Namespace, _SubParsersAction
+from dataclasses import asdict
+from datetime import datetime
+from typing import Any, Dict, List, Optional, Union
+
+import requests
+from dotenv import dotenv_values
+
+from huggingface_hub import HfApi
+from huggingface_hub.utils import logging
+
+from . import BaseHuggingfaceCLICommand
+
+
+logger = logging.get_logger(__name__)
+
+
+class JobsCommands(BaseHuggingfaceCLICommand):
+    @staticmethod
+    def register_subcommand(parser: _SubParsersAction):
+        jobs_parser = parser.add_parser("jobs", help="Commands to interact with your huggingface.co jobs.")
+        jobs_subparsers = jobs_parser.add_subparsers(help="huggingface.co jobs related commands")
+
+        # Register commands
+        InspectCommand.register_subcommand(jobs_subparsers)
+        LogsCommand.register_subcommand(jobs_subparsers)
+        PsCommand.register_subcommand(jobs_subparsers)
+        RunCommand.register_subcommand(jobs_subparsers)
+        CancelCommand.register_subcommand(jobs_subparsers)
+        UvCommand.register_subcommand(jobs_subparsers)
+
+
+class RunCommand(BaseHuggingfaceCLICommand):
+    @staticmethod
+    def register_subcommand(parser: _SubParsersAction) -> None:
+        run_parser = parser.add_parser("run", help="Run a Job")
+        run_parser.add_argument("image", type=str, help="The Docker image to use.")
+        run_parser.add_argument("-e", "--env", action="append", help="Set environment variables.")
+        run_parser.add_argument("-s", "--secret", action="append", help="Set secret environment variables.")
+        run_parser.add_argument("--env-file", type=str, help="Read in a file of environment variables.")
+        run_parser.add_argument("--secret-env-file", type=str, help="Read in a file of secret environment variables.")
```

Comment:
```suggestion
        run_parser.add_argument("-e", "--env", action="append", help="Set environment variables.")
        run_parser.add_argument("-s", "--secrets", action="append", help="Set secret environment variables.")
        run_parser.add_argument("--env-file", type=str, help="Read in a file of environment variables.")
        run_parser.add_argument("--secrets-file", type=str, help="Read in a file of secret environment variables.")
```

(nit) I'd rename to `--secrets` and `--secrets-file`